pkg/controller/hyperconverged/hyperconverged_controller_components_test.go

```diff
@@ -182,6 +182,7 @@ var _ = Describe("HyperConverged Components", func() {
 			// values we should preserve
 			outdatedResource.Data[virtconfig.FeatureGatesKey] = "old-featuregates-value-that-we-should-preserve"
 			outdatedResource.Data[virtconfig.MigrationsConfigKey] = "old-migrationsconfig-value-that-we-should-preserve"
+			outdatedResource.Data[virtconfig.DefaultNetworkInterface] = "old-defaultnetworkinterface-value-that-we-should-preserve"
```

Comment:
not sure I understand what this does. can you elaborate?

Reply:
Test that when we upgrade kubevirt,  the default network configuration will stay intact.
but thanks to you  I saw I forgot to add it's key [here](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/2d1bf98db4d20d4f8f82248f91254ad0ac078152/pkg/controller/hyperconverged/hyperconverged_controller_components_test.go#L127)

Will change that.